app/models/appointment.rb

```diff
@@ -0,0 +1,19 @@
+class Appointment < ApplicationRecord
+  belongs_to :user
```

Comment:
Depending on how far we want to take it we could just derive a `Guider` from `User`. Then the guider-specific stuff could live there at least... Hmm.